genesis/src/main.rs

```diff
@@ -201,6 +212,11 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                     bootstrap_leader_stake_lamports,
                 ),
             ),
+            // storage account
+            (
+                bootstrap_storage_keypair.pubkey(),
+                Account::new(1, 1024 * 4, &solana_storage_api::id()),
```

Comment:
did this require a change to Cargo.toml?

Reply:
Nope, got through CI fine

Reply:
what's the 4k size thing?  can you call storage_api to construct this account for you, instead?

Reply:
That's the data buffer size for the account. Yeah I'm planning to follow up with a PR that removes all uses of the magic numher.